docs/dev/cli/saucectl/run.md

```diff
@@ -28,22 +28,22 @@ Execute tests according to the environment, framework, and test suite specificat
   </thead>
```

Comment:
**For line 17, should the link be...** 

```suggestion
[configuration file](/dev/cli/saucectl/configure)
```

Reply:
Hey @kimsauce -- this is actually not correct. It's not intuitive at all, but the `configure` command actually does NOT generate the configuration file -- it generates a credentials file. This is correct as-is, pointing to the init command doc.